packages/components/helptext/src/HelpText.tsx

```diff
@@ -0,0 +1,60 @@
+import { cx, css } from 'emotion';
+import React from 'react';
+import tokens from '@contentful/f36-tokens';
+import {
+  Box,
+  CommonProps,
+  PolymorphicComponent,
+  PolymorphicComponentProps,
+  PolymorphicComponentWithRef,
+} from '@contentful/f36-core';
+
+export interface HelpTextInternalProps extends CommonProps {
+  children: React.ReactNode;
+}
+
+export type HelpTextProps = PolymorphicComponentProps<
+  React.ElementType,
+  HelpTextInternalProps
+>;
+
+const DEFAULT_TAG = 'p';
+
+const _HelpText: PolymorphicComponentWithRef<
+  HelpTextInternalProps,
+  typeof DEFAULT_TAG
+> = (
+  { children, className, testId = 'cf-ui-help-text', ...otherProps },
+  ref,
+) => {
+  return (
+    <Box
+      as={DEFAULT_TAG}
+      data-test-id={testId}
+      ref={ref}
+      className={cx(
+        css({
```

Comment:
To have better JS & CSS separation, IMO we should move those emotion definitions to `HelpText.styles.ts`.

Reply:
sure, I can do that.
I guess it's worth discussing cause we have all possible solutions in that /components folder already :)